tests/db_tests/test_db.py

```diff
@@ -0,0 +1,60 @@
+from conftests import task_data
+
+from qhana_plugin_runner.db.models.tasks import ProcessingTask
+from qhana_plugin_runner.db.models.mutable_json import (
+    TrackedDict,
+    TrackedList,
+)
+
+
+def test_json_strorage(task_data: ProcessingTask):
+    # Test mutable json data attribute
+    assert task_data.data == dict()
+
+    # Primitive data types
```

Comment:
Please do these in individual test functions. Rule of thumb should be 1 test function tests 1 thing.